staging/src/k8s.io/apiextensions-apiserver/pkg/cmd/server/options/options.go

```diff
@@ -99,10 +101,25 @@ func (o CustomResourceDefinitionsServerOptions) Config() (*apiserver.Config, err
 		return nil, err
 	}
 
+	// TODO remove me #
+	// This the another workflow to run the apiextensions-apiserver outside of kube-apiserver
+	// If everything looks fine
+	// Refactor StorageFactoryConfig as we don't need default resource types
+	// Decouple it from the kubeapiserver package
+	storageConfig := kubeapiserver.NewStorageFactoryConfig()
```

Comment:
i doubt if the users will need the etcd overrides if they are running a standalone CRD apiserver?

Reply:
This storage is full of legacy. We probably don't want to have it here (even after moving). We have to pass through a custom override data structure.